platform/common/spinning_zsync

```diff
@@ -19,6 +19,8 @@ else
     ZSYNC_MESSAGE="Computing zsync delta"
 fi
 
+# Clear screen before start
+./fbink -c
```

Comment:
Is there a particular reason not to include that -c a couple lines down below?

Reply:
Oh, right, yeah, that was my first comment, forgot about hit this morning ;).